tango_simlib/sim_xmi_parser.py

```diff
@@ -448,6 +448,10 @@ def _get_arg_type(self, description_data):
             if arg_type in ['FloatArray', 'DoubleArray', 'StringArray']:
                 arg_type = getattr(PyTango, 'DevVar' + arg_type)
             else:
+                # Substituting the 'Int' type with 'Long'. 'DevInt' is not a supported
+                # data type in TANGO.
+                if arg_type == 'Int':
+                    arg_type = 'Long'
```

Comment:
POGO files could also have a UInt type, `<dataType xsi:type="pogoDsl:UIntType"/>`, so we probably want to substitute that with `ULong`.  Also consider the array types that you can get from commands, e.g. `<type xsi:type="pogoDsl:IntArrayType"/>` - do they need to be converted as well, or does PyTango work with them as is?